source/Public/New-GuestConfigurationPolicy.ps1

```diff
@@ -409,7 +462,13 @@ function New-GuestConfigurationPolicy
         Tag = $Tag
         IncludeVMSS = $IncludeVMSS
     }
-    $policyDefinitionContent = New-GuestConfigurationPolicyContent @policyDefinitionContentParameters
+
+    if (-not ([string]::IsNullOrWhiteSpace($ManagedIdentityResourceId) -or [string]::IsNullOrWhiteSpace($LocalContentPath)))
```

Comment:
if managed identity is not empty OR local path is empty?
so always?
I don't think local path check should be here?

Reply:
If managed identity is not empty and local path is not empty then it will work.

But yes, I don't think we need to check the local content path here. As, the negative cases are already filtered out top.